pom.xml

```diff
@@ -378,53 +377,14 @@
         <!-- Container images for various integration tests -->
         <!-- The latest version of Elasticsearch tested against by default -->
         <version.org.elasticsearch.latest>8.10.4</version.org.elasticsearch.latest>
```

Comment:
We'll need a reminder in the Dockerfile that this property needs to be updated as well...